src/index.js

```diff
@@ -79,6 +81,20 @@ function displaytasks() {
   trash.forEach((t) => {
     t.addEventListener('mousedown', () => deleteTask(t.id));
   });
+  const chbox = document.querySelectorAll('.checkbox');
+  chbox.forEach((box) => {
+    box.addEventListener('change', () => {
+      checkBox(box);
+      const ch = box.parentElement.children[1];
+      if (box.checked) {
+        ch.style.textDecoration = 'line-through';
+        ch.style.opacity = '0.5';
+      } else {
+        ch.style.textDecoration = 'none';
+        ch.style.opacity = '1';
+      }
+    });
+  });
 }
 
 displaytasks();
```

Comment:
- I guess a bug may have been introduced in your form for adding task. When I try to add and submit a todo, I get an error. Can we fix this? 🤗 

![todo-err](https://user-images.githubusercontent.com/99246359/206577060-dda33cd2-3f32-43e3-94c3-0c278f7bf40a.PNG)